src/GovUk.Education.ExploreEducationStatistics.Content.Services/RedirectsService.cs

```diff
@@ -0,0 +1,65 @@
+#nullable enable
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services;
+
+public class RedirectsService : IRedirectsService
+{
+    private readonly ContentDbContext _contentDbContext;
+
+    public RedirectsService(ContentDbContext contentDbContext)
+    {
+        _contentDbContext = contentDbContext;
+    }
+
+    public async Task<Either<ActionResult, RedirectsViewModel>> List()
+    {
+        // A redirect for a methodologyVersion that isn't published shouldn't appear in the list. A redirect becomes
+        // active once the associated methodologyVersion is published. It remains active if subsequent methodology
+        // amendment's are published. We establish this by checking against each methodologyVersion's Version.
+        var methodologyRedirects = await _contentDbContext.MethodologyRedirects
+            .Where(mr => mr.MethodologyVersion.Methodology.LatestPublishedVersion != null
+                         && mr.MethodologyVersion.Methodology.LatestPublishedVersion.Version >=
+                         mr.MethodologyVersion.Version
+            )
+            .Select(mr => new
+            {
+                RedirectSlug = mr.Slug,
+                // MethodologyVersion.Slug cannot be translated into SQL, so we do this...
+                LatestPublishedSlug = mr.MethodologyVersion.Methodology.LatestPublishedVersion!.AlternativeSlug
```

Comment:
Same as previous comments here

Reply:
I left the other instances alone so I'll leave this one alone too - prioritising avoiding having to pull every latest published methodology version out of the database.